### PR TITLE
Added descriptions to arguments for millisecond

### PR DESCRIPTION
**Pull request by Miguel-F**
Added descriptions to arguments from the Lucee documentation located here - https://docs.lucee.org/reference/functions/millisecond.html

### CODE REVIEW

File: data/en/millisecond.json [hunk 1]
```diff
@@ -5,14 +5,14 @@
 	"member": "date.millisecond([timezone])",
 	"returns": "numeric",
 	"related": [],
-	"description": "",
+	"description": "Returns the milliseconds of the referenced date.",
 	"params": [
-		{"name": "date", "description": "", "required": true, "default": "", "type": "date", "values": []},
-		{"name": "timezone", "description": "", "required": false, "default": "", "type": "string", "values": []}
+		{"name": "date", "description": "date object; for example, now()", "required": true, "default": "", "type": "date", "values": []},
```
Comment (shaedrich) [inline data/en/millisecond.json]: I'm not sure if providing an example is really needed in this place…

Comment (pfreitag) [reply data/en/millisecond.json]: Probably not necessary, but it's not too wordy so I'm fine with it.
